src/session/session_api.c

```diff
@@ -2304,7 +2284,7 @@ __open_session(WT_CONNECTION_IMPL *conn, WT_EVENT_HANDLER *event_handler, const
   WT_SESSION_IMPL **sessionp)
 {
     static const WT_SESSION
-      stds = {NULL, NULL, __session_close, __session_reconfigure, __session_flush_tier,
+      stds = {NULL, NULL, __session_close, __session_reconfigure, __session_flush_tier_readonly,
```

Comment:
Just so I understand, if someone calls flush_tier now, we return unsupported?  That seems like a reasonable transition step.